### DISCUSSION TITLE
Policy not working

### DISCUSSION THREAD

**Discussion by JoulesDotDev** [supabase/supabase#8151] [Questions] [answered]
Yes, I've seen other discussion about this, and I did it how they did it, but it just won't wory
Trying to run this policy

`create policy "Allow users to read the groups they're in"
on public.note_group
for select
using ( 
   (
      auth.uid() in ( 
         SELECT user_id
         FROM note_group_membership
         WHERE note_group_id = id
      )
   )
)`

![image](https://user-images.githubusercontent.com/73104228/183463749-2c3efa06-e5ea-4cb8-bd80-13dd95614ff5.png)

I have those 2 tables
note_group has one entry with its ID
and note_group_membership has a note_group_id and a user_id
they're both correct, the sub query even returns true when isolated, but it won't work inside of the policy.

**Comment by GaryAustin1** [comment, marked as answer]
Do you have RLS on note_group_membership (in particular a select policy that will work for your user)?

**Reply by JoulesDotDev** [reply]
it makes so much sense that they're connected, I got it now, thank you.

**Reply by JoulesDotDev** [reply]
I was missing RLS on the other table, solved
